cpp/log/cert.cc

```diff
@@ -583,8 +583,8 @@ util::Status Cert::OctetStringExtensionData(int extension_nid,
   // Callers don't care whether extension is missing or invalid as they
   // usually call this method after confirming it to be present.
   const StatusOr<void*> ext_struct = ExtensionStructure(extension_nid);
-  if (!ext_struct.ok()
-      && ext_struct.status().CanonicalCode() == Code::NOT_FOUND) {
+  if (!ext_struct.ok() &&
```

Comment:
I prefer it the old way. Apparently we can ask clang-format to do:

BreakBeforeBinaryOperators: BOS_NonAssignment